recipes/matplotlib-corefonts/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "matplotlib-corefonts" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pushcx/corefonts/archive/master.zip
+  sha256: 7d3c7abe6c6f21e1a48fa1cafccb1005dcfcff9ad2f135ae219854ace6be42d0
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - cabextract
```

Comment:
There will be a race condition in the CIs when submitting both recipes together.
As long as you are fine restart the `matplotlib-corefonts` CI later this is fine. Otherwise send a separate PRs.